source/concepts/faq.md

```diff
@@ -46,3 +46,27 @@ There are downsides to relying on [experimental features](https://nixos.org/manu
   When using flakes, you will have to rely more heavily on user-to-user support, third-party documentation, and the source code.
+
+## What channels are available and what are some of the nixpkgs branches on GitHub for?
+
+There are four channels: stable, unstable, large and small.
+
+- Stable channels, e.g. [`nixos-23.05`](https://releases.nixos.org/nixos/23.05/nixos-23.05.4076.8a4c17493e5c), provide updates for fixing bugs and security vulnerabilities, but do not receive major updates after their initial release.
+New stable channels are released every six months.
+- Unstable channels, e.g. [`nixos-unstable`](https://releases.nixos.org/nixos/unstable/nixos-23.11pre531102.fdd898f8f79e) or [`nixpkgs-unstable`](https://releases.nixos.org/nixpkgs/nixpkgs-23.11pre532226.7131f3c223a2), deliver the latest tested updates on a rolling basis.
+They are not recommended for production systems.
+- Large channels, e.g. `nixos-23.05`, `nixos-unstable`, provide binary builds for the full breadth of {term}`nixpkgs`.
+- Small channels, e.g. [`nixos-23.05-small`](https://releases.nixos.org/nixos/23.05-small/nixos-23.05.4080.e49c28b3baa3), [`nixos-unstable-small`](https://releases.nixos.org/nixos/unstable-small/nixos-23.11pre532296.349bdd9653c4) contain fewer binaries than large channels.
+They get updated faster, but require more packages to be built from source than other channels. Intended for servers running {term}`NixOS`.
+
+Nix channels are reified as git branches in the [nixpkgs](https://github.com/NixOS/nixpkgs) repository. These branches have a relationship to channels, but that relationship is not always 1:1. In general:
+
+- The main development branch, `master`, is used as the starting point for other unstable channels.
+- `nix(os|pkgs)-unstable` are based on the `master` branch and contain rolling updates for both NixOS|Nix-as-a-Package-Manager (NaaPM).
+- `release-YY.MM` branches are used for stable channels as merge targets for backports.
+- Branches like `nixos-YY.MM` or `nixpkgs-YY.MM` point to a successful commit from the corresponding `release-YY.MM` branch. Used for stable channels.
+- `staging` and `staging-YY.MM` are used for mass rebuilds (number of rebuilds > 500) of the `master` and the `release-YY.MM` branches respectively.
+- `staging-next` and `staging-next-YY.MM` receive direct commits in order to fix [Hydra](https://github.com/NixOS/hydra) builds. 
+- Branches with the `*-small` suffix are the curated subsets of their larger counterparts.
+
+See the [Branch conventions](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#branch-conventions) and [Flow of merged pull requests](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#flow-of-merged-pull-requests) sections in the nixpkg's contributing guidelines for further information on branches.
```

Comment:
```suggestion
Nix channels are reified as git branches in the [nixpkgs](https://github.com/NixOS/nixpkgs) repository.

These branches have a relationship to channels, but that relationship is not always 1:1.

Consult the [Nixpkgs contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#branch-conventions) for more information on the Nixpkgs branching strategy.
```

Might be good to just replace everything below with this, or something like it. Also we should stick to the one-sentence-per-line rule in the style guide.